recipes/sire/template.yaml

```diff
@@ -0,0 +1,41 @@
+package:
```

Comment:
Please, remove this ``template.yaml``

Reply:
As mentioned in the pull request description, the meta.yaml recipe is generated dynamically during our Azure Pipelines build. I'm happy to move the template elsewhere (into the Sire repository) if it causes problems here. (It just seemed the easiest and clearest place to put it.)